conf/pdf/templates/common/styles.xsl

```diff
@@ -97,7 +104,17 @@
     <xsl:attribute name="font-size">12pt</xsl:attribute>
 </xsl:attribute-set>
 
-<xsl:attribute-set name="declaration">
+<xsl:attribute-set name="cell--heavy">
+    <xsl:attribute name="padding-top">8px</xsl:attribute>
```

Comment:
This should just extend the cell style. So cell--heavy should only contain the styles required to make it different (the two font styles).